test/gtest/uct/ib/test_ud.cc

```diff
@@ -961,19 +962,19 @@ UCS_TEST_SKIP_COND_P(test_ud_peer_failure, send_am_after_kill,
                      "UD_TIMER_TICK?=2s") {
     ucs_status_t status;
 
-    connect();
-
-    scoped_log_handler slh(wrap_errors_logger);
     flush();
-    EXPECT_EQ(0, m_err_count);
 
     status = uct_ep_check(m_e1->ep(0), 0, NULL);
     ASSERT_UCS_OK(status);
     flush();
-    /* Allow keepalive request to complete */
+    // Allow keepalive request to complete */
     short_progress_loop();
 
-    /* We are still alive */
+    /* Set TX window to big enough value to not get UCS_ERR_NO_RESOURCE from
+     * AM SHORT operations */
+    set_tx_win(m_e1, 1024);
+
+    // We are still alive */
```

Comment:
mix of c/cpp style comments in same function, pls select one style

Reply:
done